### PR TITLE
rpm: Add rpm spec file and support with Makefile

### PR DESCRIPTION
**Pull request by sc108-lee**

### CODE REVIEW

File: meson.build [hunk 1]
```diff
@@ -107,6 +107,16 @@ if is_windows
   conf_inc += [ include_directories('third-party'/'windows') ]
 endif
 
+rpm_conf = configuration_data()
+rpm_conf.set('NAME',    meson.project_name())
+rpm_conf.set('VERSION', meson.project_version())
+rpm_conf.set('LICENSE', meson.project_license())
```
Comment (karlowich) [inline meson.build:113]: This line seems to be the one breaking the tests.
`meson.project_license()` returns an array of strings and the `rpm_conf.set()` expects a single string.

Comment (eli-schwartz) [reply meson.build:113]: Having more than one is deprecated. You could index into it and only use the first one, safe in the assumption that there's no additional information being dropped.

Comment (sc108-lee) [reply meson.build]: @karlowich @eli-schwartz Thanks for the comment. 
I just added using first index of license

**Review by safl — APPROVED** [review]
Thanks @sc108-lee !

**Pull request merged**